src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1172,8 +1174,7 @@ internal async ValueTask<NpgsqlDataReader> ExecuteReader(CommandBehavior behavio
                                 try
                                 {
                                     cmd.Cancel(true);
-                                    if (cn.Settings.CancellationTimeout > 0)
-                                        cn.CommandCts.CancelAfter(cn.Settings.CancellationTimeout * 1000);
```

Comment:
Here's the bug where I completely removed the timeout from NpgsqlConnector.CommandCts.